src/simulators/state_chunk.hpp

```diff
@@ -143,7 +143,7 @@ class StateChunk : public State<state_t> {
   // Return the expectation value of a N-qubit Pauli operator
   // If the simulator does not support Pauli expectation value this should
   // raise an exception.
-  double expval_pauli(const reg_t &qubits,const std::string& pauli) override final {return 0.0;}
+  double expval_pauli(const reg_t &qubits,const std::string& pauli) override final {return expval_pauli(0, qubits, pauli);}
```

Comment:
This seems like an unrelated bugfix?  If it _is_ a bugfix, it'd be good to add a test and a release note about it.

Reply:
This is not a bug in existing usages.

`expval_pauli` should not be enabled in `AerStatevector` in this PR. I would like to revert codes related to `expval_pauli`.